model/team_member.go

```diff
@@ -32,6 +32,12 @@ type TeamMemberForExport struct {
 	TeamName string
 }
 
+type TeamMemberWithError struct {
+	UserId string
```

Comment:
since this gets passed down to JS, should this be tagged as well `json:"user_id"`? otherwise this field will get marshalled as `UserId` in json.